cdci_data_analysis/pytest_fixtures.py

```diff
@@ -480,7 +480,7 @@ def dispatcher_test_conf_with_renku_options_fn(dispatcher_test_conf_fn):
             f.write(f_default.read())
 
         f.write('\n    renku_options:'
-                '\n        renku_gitlab_repository_url: "git@renkulab.io:gabriele.barni/test-dispatcher-endpoint.git"'
+                '\n        renku_gitlab_repository_url: "git@renkulab.io:gabriele.barni/old-test-dispatcher-endpoint.git"'
```

Comment:
did you test with both new and old repositories? maybe no need to add since it can not be now done in CI anyway, just to be sure?